pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md

```diff
@@ -560,4 +561,47 @@ Each basic element can have specific style rules defined for it, controlling how
 }
 ```
 
-For a list of Rich Messaging capabilities for each channel, browse or search the table on the [Knowledge Center](https://knowledge.liveperson.com/messaging-channels-messaging-channels-capabilities-comparison.html).
\ No newline at end of file
+### Element Accessibility Attributes
```

Comment:
Please switch to sentence case.

Reply:
@j9t same with the sentence case. I see non sentence case in other places too. So I thought this was the practice.

Reply:
Fair enough—we can work with this for now. It’s still widespread on the site, but we are switching to sentence case; see https://github.com/LivePersonInc/developer-center#notes-on-content-and-code for details.